### ISSUE TITLE
Documentation for Windows Install Missing Information

### ISSUE THREAD

**Issue by LeaderOfChampions** [input-output-hk/shelley-testnet#41]
When I get to step #6 for installing with Windows (See: https://github.com/input-output-hk/shelley-testnet/wiki/How-to-setup-a-Jormungandr-Networking--node-%28v-0.5.0%29) it failed/isn't clear what to do.

How to Install Jormungandr on Windows?

Download version 0.5 of jormungandr. https://github.com/input-output-hk/jormungandr/releases/
**I downloaded the latest version which is 0.5.2**

Select the jormungandr-v0.5.0-x86_64-pc-windows-gnu.zip
**Did this without a problem.**

Open PowerShell as Administrator
**Straight forward**

Decompress the file: expand-archive jormungandr-v0.5.0-x86_64-pc-windows-gnu.zip
**Worked just fine, note I did also try just unzipping via Windows**

Then you can list the files: dir
**Sure, okay, done...**

Then download the configuration file from: node-config.yaml Now you can execute the node

**Okay, here is where I got lost. I tried making a file with NotePad++ and renaming it to node-config.yaml with the following pasted inside the file:**

--- 
log: 
  format: plain
  level: debug
  output: stderr
p2p: 
  listen_address: /ip4/0.0.0.0/tcp/3100
  public_address: 
  topics_of_interest: 
    blocks: high
    messages: high
  trusted_peers: 
    - /ip4/3.123.177.192/tcp/3000
    - /ip4/52.57.157.167/tcp/3000
    - /ip4/3.123.155.47/tcp/3000
    - /ip4/3.115.57.216/tcp/3000
    - /ip4/3.112.185.217/tcp/3000
    - /ip4/18.139.40.4/tcp/3000
    - /ip4/18.140.134.230/tcp/3000
rest: 
  listen: "127.0.0.1:3101"
storage: /tmp/jormungandr

**HOWEVER, step 6 or the below command, does nothing**

jormungandr --config node-config.yaml 
--genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201 
--log-level=info
1.Then you will see node running.

**Below is the error I get from running the above after guessing about the node-config.yaml file.**
******************************************

PS D:\jormungandr-v0.5.2-x86_64-pc-windows-gnu> jormungandr --config node-config.yaml
>> --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201
>> --log-level=info                                                                                                     At line:2 char:3
+ --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b377 ...
+   ~
Missing expression after unary operator '--'.
At line:2 char:3
+ --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b377 ...
+   ~~~~~~~~~~~~~~~~~~
Unexpected token 'genesis-block-hash' in expression or statement.
At line:3 char:3
+ --log-level=info
+   ~
Missing expression after unary operator '--'.
At line:3 char:3
+ --log-level=info
+   ~~~~~~~~~~~~~~
Unexpected token 'log-level=info' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MissingExpressionAfterOperator

PS D:\jormungandr-v0.5.2-x86_64-pc-windows-gnu>                                                                                                                           

Please help.

**Comment by WoodDesk** [comment]
> Then download the configuration file from: node-config.yaml
I also got lost at this point

**Comment by WoodDesk** [comment]
I see that I got lost because the info that I needed was listed in the linux instructions located above the windows instructions.  it would be helpful to repeat the instructions from the linux section into the windows section as appropriate.

> Create the configuration file: node-config.yaml in any plain text editor and save it to the same folder as Jormungandr and jcli

I did the above in a folder for the newer version as well as for the older version of the testnet:

-  jormungandr-v0.5.0-x86_64-pc-windows-gnu

-  jormungandr-v0.5.2-x86_64-pc-windows-gnu

after doing the above i was able to proceed further using this command.  note this could not be copy/pasted from the instructions into powershell without encountering an error because the command is on 3 lines in the instructions which causes problems pasting into powershell.  here is the command I used:
`.\jormungandr --config node-config.yaml --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201 --log-level=info`


- **The result from running that command in the jormungandr-v0.5.0-x86_64-pc-windows-gnu folder:**

> PS E:\testnet\jormungandr-v0.5.0-x86_64-pc-windows-gnu> .\jormungandr --config node-config.yaml --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201 --log-level=info
> Sep 27 00:20:12.759 INFO Starting jormungandr 0.5.0 (HEAD-98922c77, release, windows [x86_64]) - [rustc 1.37.0 (eae3437df 2019-08-13)], task: init
> Sep 27 00:20:12.762 WARN Node started without path to the stored secret keys (not a stake pool or a BFT leader), task: init
> Sep 27 00:20:12.763 INFO storing blockchain in '"/tmp/jormungandr\\blocks.sqlite"', task: init
> Sep 27 00:20:12.765 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201 from 3.123.177.192:3000, task: init
> Error fetching the genesis block from the network
> no blocks in the stream


- **The result from running that command in the jormungandr-v0.5.2-x86_64-pc-windows-gnu folder:**

> PS E:\testnet\jormungandr-v0.5.2-x86_64-pc-windows-gnu>  .\jormungandr --config node-config.yaml --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201 --log-level=info
> Sep 27 00:19:11.991 INFO Starting jormungandr 0.5.2 (HEAD-e7a3efe7, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)], task: init
> Sep 27 00:19:11.994 WARN Node started without path to the stored secret keys (not a stake pool or a BFT leader), task: init
> Sep 27 00:19:12.013 INFO storing blockchain in '"/tmp/jormungandr\\blocks.sqlite"', task: init
> Sep 27 00:19:12.189 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 3.123.177.192:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:12.299 ERRO error downloading block, reason: no blocks in the stream, peer_address: 3.123.177.192:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:12.302 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 3.123.155.47:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:12.399 ERRO error downloading block, reason: no blocks in the stream, peer_address: 3.123.155.47:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:12.403 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 3.112.185.217:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:13.009 ERRO error downloading block, reason: no blocks in the stream, peer_address: 3.112.185.217:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:13.012 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 3.115.57.216:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:13.676 ERRO error downloading block, reason: block request failed, peer_address: 3.115.57.216:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:13.680 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 18.139.40.4:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:14.111 ERRO error downloading block, reason: no blocks in the stream, peer_address: 18.139.40.4:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:14.114 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 18.140.134.230:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:14.530 ERRO error downloading block, reason: no blocks in the stream, peer_address: 18.140.134.230:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:14.533 INFO fetching block 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, peer_address: 52.57.157.167:3000, block: 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201, task: init
> Sep 27 00:19:14.650 ERRO error downloading block, reason: block request failed, peer_addressError fetching the genesis block from the network
> :  |-> could not download block hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201
> 52.57.157.167:3000, block

**Comment by LeaderOfChampions** [comment]
Thank you, I believe you are correct, the copy/paste was failing. I copy-pasted from your above note and it worked like a charm, thank you.

FIX = .\jormungandr --config node-config.yaml --genesis-block-hash 1f077794039a53309576b35dcd1121375d548db0aeb0b3770a7956cba1a44201 --log-level=info

**Comment by elocremarc** [comment]
> - The result from running that command in the jormungandr-v0.5.2-x86_64-pc-windows-gnu folder:

@WoodDesk I got the same error messages when running the command for V0.5.2
>ERRO error downloading block, reason: no blocks in the stream, peer_address: 18.139.40.4:3000

@LeaderOfChampions  did you get the same result or did you get it working w/o errors?

**Comment by mattfanz** [comment]
Addition wrong link : 
`https://testnet.iohkdev.io/cardano/shelley/faucet/` should be : `https://testnet.iohkdev.io/shelley/tools/faucet/`

`jcli rest v0 account get $(cat receiver_account.txt) -h  http://127.0.0.1:8443/api` should be :
`jcli rest v0 account get $(cat receiver_account.txt) -h  http://127.0.0.1:3101/api`

**Comment by LeaderOfChampions** [comment]
@elocremarc Honestly I'm still lost, I get errors about:
`ERRO error downloading block, reason: block request failed, peer_address:Error fetching the genesis block from the network`

**Comment by lazrfocus** [comment]
i get a slightly different error, running from windows 10.

` ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 18.139.40.4:3000, task: bootstrap`

happens for all peers.
it appears i can connect, though...

```
Sep 26 20:21:36.623 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 18.139.40.4:3000, task: bootstrap
Sep 26 20:21:36.626 INFO connecting to bootstrap peer 52.57.157.167:3000, peer_addr: 52.57.157.167:3000, task: bootstrap
Sep 26 20:21:36.628 DEBG connecting to 52.57.157.167:3000, scope: global
Sep 26 20:21:36.629 DEBG adding I/O source: 0, scope: global
Sep 26 20:21:36.630 DEBG scheduling Write for: 0, scope: global
Sep 26 20:21:36.783 DEBG connected to Some(V4(52.57.157.167:3000)), scope: global
Sep 26 20:21:36.785 DEBG binding client connection, scope: global
Sep 26 20:21:36.786 DEBG client connection bound, scope: global
Sep 26 20:21:36.788 DEBG send; frame=Settings { flags: (0x0), enable_push: 0 }, scope: global
Sep 26 20:21:36.789 DEBG pulling blocks starting from 6260bcf5d23cf3b35500614556c74490793e6969ea272f4160ff3a1bc8bd6383, peer_addr: 52.57.157.167:3000, task: bootstrap
```

i dont think its pulling any blocks, but in the debug log every few seconds its scheduling a read, so its trying to do stuff, but i havent bootstrapped yet.

```
Sep 26 20:28:37.597 DEBG scheduling Read for: 2, scope: global
Sep 26 20:28:37.598 DEBG scheduling Read for: 0, scope: global
Sep 26 20:28:37.600 DEBG scheduling Read for: 0, scope: global
Sep 26 20:28:37.601 DEBG scheduling Read for: 0, scope: global
Sep 26 20:28:37.602 DEBG scheduling Read for: 3, scope: global
Sep 26 20:28:37.603 DEBG scheduling Read for: 1, scope: global
Sep 26 20:28:37.604 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
Sep 26 20:28:37.606 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
Sep 26 20:28:37.607 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
Sep 26 20:28:37.609 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
Sep 26 20:28:37.610 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
Sep 26 20:28:37.611 DEBG scheduling Read for: 1, scope: global
Sep 26 20:28:37.612 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
Sep 26 20:28:37.613 DEBG send; frame=Data { stream_id: StreamId(7) }, scope: global
```

**Comment by lazrfocus** [comment]
@LeaderOfChampions 

your genesis block hash is wrong in your command.  try the new one:
`adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770`

**Comment by LeaderOfChampions** [comment]
@lazrfocus 

Thanks, when I put in your mentioned hash the following happens:

Note, I've tried msvc and gru (honestly not clear on the difference)

`PS D:\jormungandr-v0.5.2-x86_64-pc-windows-msvc> .\jormungandr --config node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770 --log-level=info
Sep 26 21:42:12.124 INFO Starting jormungandr 0.5.2 (HEAD-e7a3efe7, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)], task: init
Sep 26 21:42:12.128 WARN Node started without path to the stored secret keys (not a stake pool or a BFT leader), task: init
Sep 26 21:42:12.135 INFO storing blockchain in '"/tmp/jormungandr\\blocks.sqlite"', task: init
Sep 26 21:42:14.292 INFO connecting to bootstrap peer 18.139.40.4:3000, peer_addr: 18.139.40.4:3000, task: bootstrap
Sep 26 21:42:14.728 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 18.139.40.4:3000, task: bootstrap
Sep 26 21:42:14.735 INFO connecting to bootstrap peer 18.140.134.230:3000, peer_addr: 18.140.134.230:3000, task: bootstrap
Sep 26 21:42:15.154 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 18.140.134.230:3000, task: bootstrap
Sep 26 21:42:15.162 INFO connecting to bootstrap peer 3.123.177.192:3000, peer_addr: 3.123.177.192:3000, task: bootstrapSep 26 21:42:15.495 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.123.177.192:3000, task: bootstrap
Sep 26 21:42:15.498 INFO connecting to bootstrap peer 3.123.155.47:3000, peer_addr: 3.123.155.47:3000, task: bootstrap
Sep 26 21:42:16.406 INFO initial bootstrap completed, peer_addr: 3.123.155.47:3000, task: bootstrap
Sep 26 21:42:16.413 INFO starting task, task: client-query
Sep 26 21:42:16.419 INFO starting task, task: network
Sep 26 21:42:16.422 INFO adding P2P Topology module: trusted-peers, task: network
Sep 26 21:42:16.429 INFO start listening and accepting gRPC connections on 0.0.0.0:3100, task: network
Sep 26 21:42:16.432 INFO Starting 1 workers, scope: global
Sep 26 21:42:16.437 INFO Starting server on 127.0.0.1:3101, scope: global
Sep 26 21:42:16.442 INFO connecting to initial gossip peer, peer_addr: 3.112.185.217:3000, task: network
Sep 26 21:42:16.450 INFO connecting to initial gossip peer, peer_addr: 3.115.57.216:3000, task: network
Sep 26 21:42:16.455 INFO connecting to initial gossip peer, peer_addr: 52.57.157.167:3000, task: network
Sep 26 21:42:16.462 INFO connecting to initial gossip peer, peer_addr: 18.139.40.4:3000, task: network
Sep 26 21:42:16.471 INFO connecting to initial gossip peer, peer_addr: 3.123.177.192:3000, task: network
Sep 26 21:42:16.477 INFO connecting to initial gossip peer, peer_addr: 3.123.155.47:3000, task: network
Sep 26 21:42:16.484 INFO connecting to initial gossip peer, peer_addr: 18.140.134.230:3000, task: network`

**Comment by lazrfocus** [comment]
thats the same step im at on windows.

ive done the same steps on Ubuntu WSL (full ubuntu terminal on windows 10) with the binary and the node is up and running...no luck on windows itself.

**Comment by elviejo79** [comment]
Thank you @WoodDesk  and @mattfanz I've implemented your corrections in the documentation.

**Comment by elviejo79** [comment]
@LeaderOfChampions now that you got the node installed.
Can I close this ticket and if you continue to experiencing networking issues,
report them on a new issue?

**Comment by Jerjon89** [comment]
@elviejo79 , you mention that LeaderOfChampions's node is installed.
But is it running as intended? I also see the same "Error with initial bootstrap" messages appear.
Same issue thus as @LeaderOfChampions 

Will these error's cause issues on future steps? Do they have to be resolved?
If so, please share how.


PS C:\Users\*\Documents\gitprojects\jormungandr-node> .\jormungandr --config node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770 --log-level=info
Sep 27 12:00:53.008 INFO Starting jormungandr 0.5.2 (HEAD-e7a3efe7, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)], task: init
Sep 27 12:00:53.010 WARN Node started without path to the stored secret keys (not a stake pool or a BFT leader), task: init
Sep 27 12:00:53.012 INFO storing blockchain in '"/tmp/jormungandr\\blocks.sqlite"', task: init
Sep 27 12:00:55.600 INFO connecting to bootstrap peer 3.123.177.192:3000, peer_addr: 3.123.177.192:3000, task: bootstrapSep 27 12:00:55.661 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.123.177.192:3000, task: bootstrap
Sep 27 12:00:55.669 INFO connecting to bootstrap peer 18.139.40.4:3000, peer_addr: 18.139.40.4:3000, task: bootstrap
Sep 27 12:00:56.216 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 18.139.40.4:3000, task: bootstrap
Sep 27 12:00:56.223 INFO connecting to bootstrap peer 18.140.134.230:3000, peer_addr: 18.140.134.230:3000, task: bootstrap
Sep 27 12:00:56.781 ERRO with initial bootstrap, reason: bootstrap pull stream failed, peer_addr: 18.140.134.230:3000, task: bootstrap
Sep 27 12:00:56.789 INFO connecting to bootstrap peer 3.112.185.217:3000, peer_addr: 3.112.185.217:3000, task: bootstrapSep 27 12:00:57.334 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.112.185.217:3000, task: bootstrap
Sep 27 12:00:57.342 INFO connecting to bootstrap peer 52.57.157.167:3000, peer_addr: 52.57.157.167:3000, task: bootstrapSep 27 12:00:57.392 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 52.57.157.167:3000, task: bootstrap
Sep 27 12:00:57.399 INFO connecting to bootstrap peer 3.123.155.47:3000, peer_addr: 3.123.155.47:3000, task: bootstrap
Sep 27 12:00:57.468 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.123.155.47:3000, task: bootstrap
Sep 27 12:00:57.475 INFO connecting to bootstrap peer 3.115.57.216:3000, peer_addr: 3.115.57.216:3000, task: bootstrap
Sep 27 12:00:58.046 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.115.57.216:3000, task: bootstrap
Sep 27 12:00:58.053 INFO starting task, task: client-query
Sep 27 12:00:58.058 INFO starting task, task: network
Sep 27 12:00:58.061 INFO adding P2P Topology module: trusted-peers, task: network
Sep 27 12:00:58.063 INFO start listening and accepting gRPC connections on 0.0.0.0:3100, task: network
Sep 27 12:00:58.067 INFO Starting 1 workers, scope: global
Sep 27 12:00:58.069 INFO Starting server on 127.0.0.1:3101, scope: global
Sep 27 12:00:58.072 INFO connecting to initial gossip peer, peer_addr: 3.112.185.217:3000, task: network
Sep 27 12:00:58.077 INFO connecting to initial gossip peer, peer_addr: 3.115.57.216:3000, task: network
Sep 27 12:00:58.083 INFO connecting to initial gossip peer, peer_addr: 52.57.157.167:3000, task: network
Sep 27 12:00:58.088 INFO connecting to initial gossip peer, peer_addr: 18.139.40.4:3000, task: network
Sep 27 12:00:58.093 INFO connecting to initial gossip peer, peer_addr: 3.123.177.192:3000, task: network
Sep 27 12:00:58.100 INFO connecting to initial gossip peer, peer_addr: 3.123.155.47:3000, task: network
Sep 27 12:00:58.107 INFO connecting to initial gossip peer, peer_addr: 18.140.134.230:3000, task: network

**Comment by Jerjon89** [comment]
> @elviejo79 , you mention that LeaderOfChampions's node is installed.
> But is it running as intended? I also see the same "Error with initial bootstrap" messages appear.
> Same issue thus as @LeaderOfChampions
> 
> Will these error's cause issues on future steps? Do they have to be resolved?
> If so, please share how.
> 
> PS C:\Users*\Documents\gitprojects\jormungandr-node> .\jormungandr --config node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770 --log-level=info
> Sep 27 12:00:53.008 INFO Starting jormungandr 0.5.2 (HEAD-e7a3efe7, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)], task: init
> Sep 27 12:00:53.010 WARN Node started without path to the stored secret keys (not a stake pool or a BFT leader), task: init
> Sep 27 12:00:53.012 INFO storing blockchain in '"/tmp/jormungandr\blocks.sqlite"', task: init
> Sep 27 12:00:55.600 INFO connecting to bootstrap peer 3.123.177.192:3000, peer_addr: 3.123.177.192:3000, task: bootstrapSep 27 12:00:55.661 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.123.177.192:3000, task: bootstrap
> Sep 27 12:00:55.669 INFO connecting to bootstrap peer 18.139.40.4:3000, peer_addr: 18.139.40.4:3000, task: bootstrap
> Sep 27 12:00:56.216 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 18.139.40.4:3000, task: bootstrap
> Sep 27 12:00:56.223 INFO connecting to bootstrap peer 18.140.134.230:3000, peer_addr: 18.140.134.230:3000, task: bootstrap
> Sep 27 12:00:56.781 ERRO with initial bootstrap, reason: bootstrap pull stream failed, peer_addr: 18.140.134.230:3000, task: bootstrap
> Sep 27 12:00:56.789 INFO connecting to bootstrap peer 3.112.185.217:3000, peer_addr: 3.112.185.217:3000, task: bootstrapSep 27 12:00:57.334 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.112.185.217:3000, task: bootstrap
> Sep 27 12:00:57.342 INFO connecting to bootstrap peer 52.57.157.167:3000, peer_addr: 52.57.157.167:3000, task: bootstrapSep 27 12:00:57.392 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 52.57.157.167:3000, task: bootstrap
> Sep 27 12:00:57.399 INFO connecting to bootstrap peer 3.123.155.47:3000, peer_addr: 3.123.155.47:3000, task: bootstrap
> Sep 27 12:00:57.468 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.123.155.47:3000, task: bootstrap
> Sep 27 12:00:57.475 INFO connecting to bootstrap peer 3.115.57.216:3000, peer_addr: 3.115.57.216:3000, task: bootstrap
> Sep 27 12:00:58.046 ERRO with initial bootstrap, reason: bootstrap pull request failed, peer_addr: 3.115.57.216:3000, task: bootstrap
> Sep 27 12:00:58.053 INFO starting task, task: client-query
> Sep 27 12:00:58.058 INFO starting task, task: network
> Sep 27 12:00:58.061 INFO adding P2P Topology module: trusted-peers, task: network
> Sep 27 12:00:58.063 INFO start listening and accepting gRPC connections on 0.0.0.0:3100, task: network
> Sep 27 12:00:58.067 INFO Starting 1 workers, scope: global
> Sep 27 12:00:58.069 INFO Starting server on 127.0.0.1:3101, scope: global
> Sep 27 12:00:58.072 INFO connecting to initial gossip peer, peer_addr: 3.112.185.217:3000, task: network
> Sep 27 12:00:58.077 INFO connecting to initial gossip peer, peer_addr: 3.115.57.216:3000, task: network
> Sep 27 12:00:58.083 INFO connecting to initial gossip peer, peer_addr: 52.57.157.167:3000, task: network
> Sep 27 12:00:58.088 INFO connecting to initial gossip peer, peer_addr: 18.139.40.4:3000, task: network
> Sep 27 12:00:58.093 INFO connecting to initial gossip peer, peer_addr: 3.123.177.192:3000, task: network
> Sep 27 12:00:58.100 INFO connecting to initial gossip peer, peer_addr: 3.123.155.47:3000, task: network
> Sep 27 12:00:58.107 INFO connecting to initial gossip peer, peer_addr: 18.140.134.230:3000, task: network

EDIT: Issue is discussed and clarified in other topic it seems: https://github.com/input-output-hk/shelley-testnet/issues/42

**Comment by LeaderOfChampions** [comment]
@elviejo79 

Thank you very much. The documentation is clear now.
